Trappist/src/Promact.Trappist.DomainModel/Models/Test/Test.cs

```diff
@@ -33,6 +31,7 @@ public class Test : BaseModel
         public virtual ICollection<TestAttendees> TestAttendees { get; set; }
         public virtual ICollection<TestQuestion> TestQuestion { get; set; }
         public virtual ICollection<TestCategory> TestCategory { get; set; }
+        public virtual ICollection<TestIpAddress> TestIPAddress { get; set; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename property 'TestIPAddress' to match camel case naming rules, consider using 'TestIpAddress'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)